third_party/move/move-compiler-v2/tests/checking/typing/lambda2.move

```diff
@@ -0,0 +1,100 @@
+module 0x8675309::M {
+    // use 0x1::XVector;
+
+    // public inline fun foreach<T>(v: &vector<T>, action: |&T|) { // expected to be not implemented
```

Comment:
Bunch of commented out code.

Reply:
The idea of these heavily commented-out tests is that lambda.move is the original test, with no comments.  Some errors don't show up, so we comment out the code generating errors on lambda.move, and call that lambda2.move.  That test shows certain other errors, so we comment out the relevant lines there, giving lambda3.move, etc.  I left the commented-out lines to keep the line numbers the same.  Since there is an empty line, I'll add a comment on that line to indicate what we are doing.